presto-main/src/test/java/com/facebook/presto/execution/TestTaskExecutor.java

```diff
@@ -130,6 +130,8 @@ public void testTaskHandle()
             beginPhase.register();
             Phaser verificationComplete = new Phaser();
             verificationComplete.register();
+            // register once more to make splits could not be finished
+            verificationComplete.register();
```

Comment:
I think it would be better to fix this by constructing two `TestingJob` objects. I'm not sure why this test is written calling `enqueueSplit` twice with the same `driver`, but that's not the right way to use it.